src/applications/healthcare/questionnaire/components/veteran-info/AppointmentInfoBox.jsx

```diff
@@ -123,22 +124,26 @@ const AppointmentInfoBox = ({
   );
 };
 
-const mapStateToProps = state => ({
-  userFullName: state.user?.profile?.userFullName,
-  dateOfBirth: state.user?.profile?.dob,
-  gender: state.user?.profile?.gender,
-  addresses: {
-    residential: state.user?.profile?.vet360?.residentialAddress,
-    mailing: state.user?.profile?.vet360?.mailingAddress,
-  },
-  phoneNumbers: [
-    { label: 'Home', data: state.user?.profile?.vet360?.homePhone },
-    { label: 'Mobile', data: state.user?.profile?.vet360?.mobilePhone },
-    { label: 'Work', data: state.user?.profile?.vet360?.workPhone },
-    { label: 'Temporary', data: state.user?.profile?.vet360?.temporaryPhone },
-  ],
-  appointment: state.questionnaireData?.context?.appointment,
-});
+const mapStateToProps = state => {
+  const profile = selectProfile(state);
+  const vapContactInfo = selectVAPContactInfo(state);
+  return {
+    userFullName: profile.userFullName,
```

Comment:
note: no need to do `profile?.` since the `selectProfile()` selector will return an empty object if profile data doesn't exist in Redux.